### ISSUE TITLE
all templates 2023.12 broken

### ISSUE THREAD

**Issue by fsecko1** [Vaarlion/HACS_home-assistant_template.light#1]
After update to 2023.12 all my template lights, switches, sensors are broken
After removing this bugfix, everything works fine
![HA](https://github.com/Vaarlion/HACS_home-assistant_template.light/assets/72581549/30b4612f-c63a-457c-9b65-0263a83fe949)

![HA2](https://github.com/Vaarlion/HACS_home-assistant_template.light/assets/72581549/ae3f0b75-6248-4953-868e-7fe9944733c1)

![HA3](https://github.com/Vaarlion/HACS_home-assistant_template.light/assets/72581549/b105950e-f0b4-47c6-adc3-c210c1332f3a)

**Comment by Vaarlion** [comment]
yes, I don't know how to fix this without pulling it off ... fully ... they don't seam to be a way to setup a "max version" the same way their is a min version ...

But yes, this was finally merged into HA.
